xbmc/filesystem/File.cpp

```diff
@@ -387,6 +397,21 @@ bool CFile::OpenForWrite(const CURL& file, bool bOverWrite)
   return false;
 }
 
+bool CFile::ReOpen(const CURL& file)
+{
+  if (!m_pFile)
```

Comment:
Open already has "flags" parameter wich could also be used (without any API change),
@FernetMenta pls let me know if this is an option, then I can save the time putting ReOpen at 100 places in the api